specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/compute.json

```diff
@@ -9669,6 +9672,12 @@
           "$ref": "#/definitions/DedicatedHostInstanceView",
           "readOnly": true,
           "description": "The dedicated host instance view."
+        },
+        "timeCreated": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "Specifies the time at which the Dedicated Host resource was created.<br><br>Minimum api-version: 2021-11-01."
```

Comment:
per @pilor (ARM reviewer):
this type of information should go in the common systemData contract that all resources SHOULD be supporting in any new API version. When does CRP plan to implement systemData for their resources? https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources